ops/docker/op-stack-go/Dockerfile

```diff
@@ -165,4 +165,4 @@ CMD ["da-server"]
 
 FROM --platform=$TARGETPLATFORM $TARGET_BASE_IMAGE as op-supervisor-target
 COPY --from=op-supervisor-builder /app/op-supervisor/bin/op-supervisor /usr/local/bin/
-ENTRYPOINT ["op-supervisor"]
+CMD ["op-supervisor"]
```

Comment:
Does changing this from ENTRYPOINT to CMD affect the op-supervisor container functionality?

Reply:
`op-supervisor` is not running in prod, or even testnet, yet. Changing the behavior to be consistent with the other containers is good. Thanks for that.